config/config.go

```diff
@@ -140,13 +176,40 @@ func (cfg Config) Validate() error { //nolint:gocognit
 		inputErrors = append(inputErrors, fmt.Errorf("-globalTimeout: must be > 0, we got %d", cfg.RunnerOptions.GlobalTimeout))
 	}
 
+	if !reflect.ValueOf(cfg.Request.Body).IsZero() {
+		if len(cfg.Request.Body.Content) == 0 {
+			inputErrors = append(inputErrors, fmt.Errorf("-bodyContent: you must provide a value if you have added a bodyType"))
+		}
+		if reflect.ValueOf(cfg.Request.Body.Type).IsZero() {
+			inputErrors = append(inputErrors, fmt.Errorf("-bodyType: you must provide a value if you have added a bodyContent"))
+		} else if !contains(contentTypeValidValues, cfg.Request.Body.Type) {
```

Comment:
As far as I remember, that `Type` was supposed to be an indication for us to know how we treat the content: is it raw text or is it a path to a file we should retrieve (or else)